### ISSUE TITLE
Washington State data issues[Data Issue]: 

### ISSUE THREAD

**Issue by curtzt** [electricitymaps/electricitymaps-contrib#4900]
### When did this happen?

2022:12:25 11:54AM

### What zones are affected?

I don't see how to get a zone from the website with the map display.

### What is the problem?

There's a PUD for Clark County, just north of the Portland General Electric territory on the current map. A friend of mine who knows more than I do also says that the current map puts Pacificorp West in Cowlitz County (where it owns dams, but has no load). I'm not sure where the county lines would go on the map, so I don't know if he's right or not.

**Comment by VIKTORVAV99** [comment]
Hello @curtzt!

Not really sure if I understood you right here but it seems you think that there is one zones missing for Clark County on the map and that Pacificorp West is incorrect? 

If that's the case could you just confirm that's the case?

**Comment by curtzt** [comment]
Yes…. You don’t seem to have the load for this public utility district included - it is the utility for all of Clark County, which includes Vancouver WA, just across the river from Portland.

https://www.clarkpublicutilities.com/

My friend thinks that your map shows Cowlitz County, WA, which is here -



as included in PacificorpWest’s grid. If he’s right about the map borders, he says that’s a mistake. They own dams in that county, but the load there is not part of their grid. It’s actually served by another PUD.

https://www.cowlitzpud.org/

Best wishes,
Thad

**Comment by VIKTORVAV99** [comment]
I'll look into it but it might take a while with the holidays and all, I hope you understand.

**Comment by curtzt** [comment]
I don’t need it fixed - I just thought you might like to know…

Best wishes for the new year,
Thad

**Comment by VIKTORVAV99** [comment]
@unitrium you seem more familiar with the EIA zones so I was hoping you could check if the production for Public Utility District No. 1 of Clark County is included in any of our other zones or if we are simply missing this value. It should have some wind production and possibly hydro but it's unclear where the hydro is reported since it seems to use purchase agreements with BPA and possibly PacifiCorp.
